packages/dev/sharedUiComponents/src/nodeGraphSystem/graphCanvas.tsx

```diff
@@ -349,6 +350,11 @@ export class GraphCanvasComponent extends React.Component<IGraphCanvasComponentP
         dataGenerator: (nodeData: INodeData) => any,
         rootElement: HTMLDivElement
     ) {
+        if (evt.code === "Space") {
+            this.stateManager.modalIsDisplayed = true;

```

Comment:
```suggestion
            const hostBounds = this._hostCanvas.getBoundingClientRect();
            if (mouseLocationX < hostBounds.left || mouseLocationX > hostBounds.right || mouseLocationY < hostBounds.top || mouseLocationY > hostBounds.bottom) {
                return;
            }
            this.stateManager.modalIsDisplayed = true;
```

Reply:
good catch. I wonder if it would be faster by checking the source of the event?